components/Header/header.scss

```diff
@@ -176,8 +176,8 @@ header {
   *,
   ::before,
   ::after {
-    box-sizing: border-box;  
-    font-family: var(--font-poppins), sans-serif;
+    box-sizing: border-box;
+    font-family: var(--font-inter), sans-serif;
```

Comment:
I'm wondering if we shouldn't use a global font variable for this. It would avoid needing to find out where to replace a variable in future, should font's change again for some reason.
Something like `--font-header` which in our globals would be:
 `--font-header: var(--font-inter);`

Reply:
@fraserclark can you create a separate ticket for that work?

Reply:
Yep is now
https://github.com/CDLUC3/dmsp_frontend_prototype/issues/165

Reply:
Thank you